.vscode/settings.json

```diff
@@ -23,4 +23,5 @@
 	],
 	"editor.insertSpaces": false,
 	"editor.detectIndentation": false,
+	"git.ignoreLimitWarning": true,
```

Comment:
This should be in your global vscode settings. This file is what broke my git when I was trying to checkout your branch :man_shrugging:

Reply:
Removed and updated locally. Thanks!